templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} manufacturer:</h1>
```

Comment:
The `yesno` filter is used incorrectly here. It should be applied to a boolean value to determine whether to display 'Update' or 'Create'. Ensure that `object` is a boolean or adjust the logic to correctly determine the form's purpose.